### PR TITLE
Trigger assets for 9.4.X-lts tag releases 

### PR DESCRIPTION
**Pull request by yash-zededa**
The 9.4-stable branch was missing the assets trigger. Although the trigger of assets is from master branch seems like GHA needs the trigger to be part of the release as well.

### CODE REVIEW

File: .github/workflows/publish.yml
Comment (uncleDecart) [inline .github/workflows/publish.yml:169]: I think it should use assets.yml from 9.4 tag release. Changing things in master could break backward compatibility for this or am I missing something?

Comment (yash-zededa) [reply .github/workflows/publish.yml:169]: If we refer the actions for `[11.8.0](https://github.com/lf-edge/eve/tree/11.8.0)` tag. the assets trigger was pointed to the the master branch and the trigger was successful.

image

Comment (uncleDecart) [reply .github/workflows/publish.yml:169]: What if we change the way assets are build? For instance, in master we add new UEFI file or anything actually and then workflow from 9.4 (which doesn't have this) will trigger master and will fail. Why not use local assets.yml file which is in this branch ?

Comment (yash-zededa) [reply .github/workflows/publish.yml:169]: I would prefer to keep it simple For eg. in the tag the first two digits separated by `.` will always be the branch name  . We can then append the `-stable` so that it will be standard across all the `lts` releases.

Comment (uncleDecart) [reply .github/workflows/publish.yml:169]: What I'm saying is we do 
`uses: .github/workflows/assets.yml@master`
instead of
`lf-edge/eve/.github/workflows/assets.yml@master`

So that we trigger assets.yml file which is on this specific branch and not on EVE master. That way we don't have to maintain compatibility of assets.yml across all of EVE versions

Comment (OhmSpectator) [reply .github/workflows/publish.yml:169]: I think using a local (not a master) action makes sense, as it keeps the action in sync with the way we build the package.

Comment (yash-zededa) [reply .github/workflows/publish.yml:169]: @uncleDecart this makes sense. Nor sure why I added the reference `@master` :)

Comment (yash-zededa) [reply .github/workflows/publish.yml]: @OhmSpectator I would also prefer the same. Changes are done. Once this is working as expected, I will update it for 10 and 11 stable branch.

**Review by OhmSpectator — APPROVED** [review]
It would be nice to squash the commits, but it's not critical.

**Comment by yash-zededa** [comment]
> It would be nice to squash the commits, but it's not critical.

I would squash at once waiting for @eriknordmark to approve as his approval would be required to merge this PR.

File: .github/workflows/publish.yml
Comment (yash-zededa) [reply .github/workflows/publish.yml:171]: @OhmSpectator I had made changes in the assets.yml file so that it can be triggered by publish.yml by passing the tag. I need to make changes in the assets.yml so that it accepts the input tag and then does publishes the artefacts.

Comment (OhmSpectator) [reply .github/workflows/publish.yml:171]: So, should we update the assets.yml as well in this PR?.. If this version of publish.yml expects the new version of assets.yml...

Comment (yash-zededa) [reply .github/workflows/publish.yml:171]: @OhmSpectator It's done.

**Comment by OhmSpectator** [comment]
@uncleDecart, could you take a look again, please? I hope we can use it today to make a bright new 9.4.11 release =)

**Review by uncleDecart — APPROVED** [review]
LGTM, let's merge it, but we need to fix using local assets.yml in master and backport it to other branches

File: .github/workflows/publish.yml
Comment (uncleDecart) [inline .github/workflows/publish.yml:169]: again, ./.github/workflows/assets.yml will use local one

**Review by eriknordmark — APPROVED** [review]
Let's merge this and see if it works.

**Pull request merged**
